src/lib/ipfs.ts

```diff
@@ -14,12 +15,16 @@ export default class Ipfs {
   private dirUrl: (dirHash: string) => string
   private fileUrl: (fileHash: string) => string
   private logger: Logger
+  private versionURL: string
+  private peersURL: string
 
   constructor({ host, port, logger }: { host: string; port: number; logger: Logger }) {
     this.addUrl = `http://${host}:${port}/api/v0/add?cid-version=0&wrap-with-directory=true`
     this.dirUrl = (dirHash) => `http://${host}:${port}/api/v0/ls?arg=${dirHash}`
     this.fileUrl = (fileHash) => `http://${host}:${port}/api/v0/cat?arg=${fileHash}`
     this.logger = logger.child({ module: 'ipfs' })
+    this.versionURL = `http://${env.IPFS_HOST}:${env.IPFS_PORT}/api/v0/version`
+    this.peersURL = `http://${env.IPFS_HOST}:${env.IPFS_PORT}/api/v0/swarm/peers`
```

Comment:
`IPFS_HOST` and `IPFS_PORT` are already passed to the constructor as `host` and `port`